pylintrc

```diff
@@ -380,6 +380,6 @@ ext-import-graph =
 int-import-graph = 
 
 [EXCEPTIONS]
-overgeneral-exceptions = Exception
```

Comment:
This is fine for now but for future reference, take a look at the top of this file, it shouldn't be updated manually, it's generated and should be updated using `edx-lint`